app/src/components/alarm-clock/alarm-clock.js

```diff
@@ -1,75 +1,75 @@
-import { Component } from "react";
+import { Component } from 'react'
 
-import StorageService from "../../service/storageService";
+import StorageService from '../../service/storageService'
 
 class AlarmClock extends Component {
-  constructor() {
-    super();
-    //window.Neutralino.computer.getMemoryInfo();
+  constructor () {
+    super()
+    // window.Neutralino.computer.getMemoryInfo();
 
     this.state = {
-      currentTime: "",
-      alarms: [],
-    };
+      currentTime: '',
```

Comment:
Codacy has a fix for the issue: [Replace `''` with `""`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
      currentTime: "",
```